modules/news/news.service.js

```diff
@@ -29,14 +32,14 @@ class NewsService {
   }
 
   async checkNewsExist(data) {
-    const news = await News.find({
+    const news = await News.countDocuments({
```

Comment:
Please, call variable properly, because it is not an actual entity, it is a count